src/lib/SessionStatus.svelte

```diff
@@ -29,26 +33,65 @@
 
   function checkSessionStatus() {
     var opIframe = document.getElementById('opIframe');
-    var message = `${sofClient.getClient().getState('clientId')} ${sofClient.getClient().getState('tokenResponse.session_state')}`;
+    var clientId = sofClient.getClient().getState('clientId');
+    var sessionId = sofClient.getClient().getState('tokenResponse.session_state');
+    var message = `${clientId} ${sessionId}`;
     // Send message to OP iframe
     opIframe?.contentWindow?.postMessage(message, OIDC_BASE);
   }
 
+  let backupInactivityTimer: NodeJS.Timeout | undefined;
+  function resetBackupInactivityTimer() {
+    if (backupInactivityTimer !== undefined) {
+        clearTimeout(backupInactivityTimer);
+      }
+      backupInactivityTimer = undefined;
+      backupInactivityTimer = setTimeout(() => goto('/logout'), BACKUP_INACTIVITY_TIMEOUT);
+  }
+  function onVisible_resetBackupInactivityTimer() {
+    if (document.visibilityState === "visible") {
+      onVisible_resetBackupInactivityTimer();
```

Comment:
@daniellrgn  do you mean to call `resetBackupInactivityTimer ` here instead?

Reply:
Thank you Amy! Fixed